### ISSUE TITLE
Android back button should close just the side image menu

### ISSUE THREAD

**Issue by bitcoinuser** [sepehr-safari/pinstr#9]
Hi,

Android back button should close just the side image menu.

https://github.com/sepehr-safari/pinstr/assets/1178856/f8f69ef9-15da-4565-a04a-d3e776cee1d0

**Comment by sepehr-safari** [comment]
ok thanks. just click outside and it will be closed properly cause it's just an modal overlay not a new page.

**Comment by bitcoinuser** [comment]
> ok thanks. just click outside and it will be closed properly cause it's just an modal overlay not a new page.

Yes but I use Android back button a lot, so would be great if possible to use it to close this modal, instead of click outside. Thanks.
